src/app/docs/button/toolbar-guidelines.component.html

```diff
@@ -0,0 +1,287 @@
+    </sky-definition-list>
+  </sky-column>
+</sky-row>
+
+<div class="sky-emphasized">Share button</div>
+<sky-row>
+  <sky-column screenSmall="3" screenMedium="2">
+    <div style="margin: 5px 0;"><button class="sky-btn sky-btn-default"><i class="fa fa-share" style="margin-right: 5px;"></i>Share</button></div>
+  </sky-column>
+  <sky-column screenSmall="9" screenMedium="10">
+    <sky-definition-list>
+      <sky-definition-list-content *ngFor="let item of shareButtonInfo">
+        <sky-definition-list-label>
+          {{item.label}}
+        </sky-definition-list-label>
+        <sky-definition-list-value>
+          {{item.value}}
+        </sky-definition-list-value>
+      </sky-definition-list-content>
+    </sky-definition-list>
+  </sky-column>
+</sky-row>
+
+<div class="sky-emphasized">Export button</div>
+<sky-row>
+  <sky-column screenSmall="3" screenMedium="2">
+    <div style="margin: 5px 0;"><sky-dropdown><sky-dropdown-button><i class="fa fa-file-o" style="margin-right: 5px;"></i>Export</sky-dropdown-button></sky-dropdown></div>
+    <div style="margin: 5px 0;"><button class="sky-btn sky-btn-default"><i class="fa fa-file-excel-o" style="margin-right: 5px;"></i>Export</button></div>
+    <div style="margin: 5px 0;"><button class="sky-btn sky-btn-default"><i class="fa fa-file-pdf-o" style="margin-right: 5px;"></i>Export</button></div>
+  </sky-column>
+  <sky-column screenSmall="9" screenMedium="10">
+    <sky-definition-list>
+      <sky-definition-list-content *ngFor="let item of exportButtonInfo">
+        <sky-definition-list-label>
+          {{item.label}}
+        </sky-definition-list-label>
+        <sky-definition-list-value>
+          {{item.value}}
+        </sky-definition-list-value>
+      </sky-definition-list-content>
+    </sky-definition-list>
+  </sky-column>
+</sky-row>
+
+<sky-docs-design-guideline
+heading="More actions"
+>
+<p>
+  Place less important or less-frequently used actions in a more actions menu. And when horizontal space in a toolbar is insufficient for all actions, collapse standard actions into the more actions menu.
+</p>
+
+<div class="sky-emphasized">More button</div>
+<sky-row>
+  <sky-column screenSmall="3" screenMedium="2">
+    <div style="margin: 5px 0;">
+      <sky-dropdown>
+        <sky-dropdown-button>
+          <i class="fa fa-ellipsis-h" style="margin-right: 5px;"></i> More
+        </sky-dropdown-button>
+        </sky-dropdown>
+    </div>
+  </sky-column>
+  <sky-column screenSmall="9" screenMedium="10">
+    <sky-definition-list>
+      <sky-definition-list-content *ngFor="let item of moreButtonInfo">
+        <sky-definition-list-label>
+          {{item.label}}
+        </sky-definition-list-label>
+        <sky-definition-list-value>
+          {{item.value}}
+        </sky-definition-list-value>
+      </sky-definition-list-content>
+    </sky-definition-list>
+  </sky-column>
+</sky-row>
+</sky-docs-design-guideline>
+
+<sky-docs-design-guideline
```

Comment:
I think we should limit the content to just the button pieces, rather than all the toolbar-related items.

Reply:
Which items specifically do you want? Based on the issue, I included all the button items that had icon info. Just let me know which ones you don't want, and I'll exclude them.

Reply:
I'll just delete everything after from "Find in this list" under the assumption that's what you mean.